packages/remote-config/lib/index.js

```diff
@@ -59,6 +83,41 @@ class FirebaseConfigModule extends FirebaseModule {
     };
     this._lastFetchTime = -1;
     this._values = {};
+    this._isWeb = Platform.OS !== 'ios' && Platform.OS !== 'android';
+  }
+
+  get defaultConfig() {
+    const updatedDefaultConfig = {};
+    Object.keys(this._values).forEach(key => {
+      // Need to make it an object with key and literal value. Not `Value` instance.
+      updatedDefaultConfig[key] = this._values[key].value;
+    });
+
+    return updatedDefaultConfig;
+  }
+
+  set defaultConfig(defaults) {
+    if (!isObject(defaults)) {
+      throw new Error("firebase.remoteConfig().defaultConfig: 'defaults' must be an object.");
+    }
+    // To make Firebase web v9 API compatible, we update the config first so it immediately
+    // updates defaults on the instance. We then pass to web platform to update. We do this because
```

Comment:
This is just a quibble on wording, but it's a mentality - if we work with apple/android/web we are not passing to web just because it's the modular API we present, we're wrapping any of the SDKs, so mental model is "we handle our business in the wrapping layer then hand off to SDK...", not just "web"

```suggestion
    // updates defaults on the instance. We then pass to underlying SDK to update. We do this because
```

Reply:
I didn't perturb the PR with these FWIW but still worth thinking about w.r.t. mental model :-)